### DISCUSSION TITLE
Reminder List is not working

### DISCUSSION THREAD

**Discussion by VMiyakawa** [uphy/obsidian-reminder#181] [Q&A]
Hello! I’ve been using the Reminder Plug-In ([Obsidian Reminder Plugin] (https://uphy.github.io/obsidian-reminder/)) and it’s been working well. I can set reminders using syntax, and the notifications are functioning properly. However, I’ve encountered an issue with the Show Reminders feature. I used to access the Control Panel, type “Show Reminders” (I even have a hotkey sequence for this), and a list of outstanding reminders would appear. Unfortunately, this is no longer working. Any help would be appreciated!

**Comment by ckpinguin** [comment]
I have a similar problem: Reminder pane on my windows machine remains completely blank, on my other machine (Mac) it still works fine. I guess there might be some cache I should delete?
